spec/game_spec.rb

```diff
@@ -16,6 +18,14 @@
         |__|__|__|
       C |  |  |  |
         |__|__|__|
+
+      Enter your move >
+      Invalid coordinates 'R2'
+
+      Enter your move >
+      Invalid coordinates '3PO'
+
+      Enter your move >
     GRID
   end
```

Comment:
I would expect more tests that are unit-y and focused around boundary conditions for coordinates. Like a coordinate is off the grid in any direction. Those kind of unit tests mean that you would have a separate object you could unit test.